packages/styles/src/components/badge/badge.stories.js

```diff
@@ -0,0 +1,23 @@
+export default {
+  title: 'Patterns/Badges, Chips, and Tags',
+  argTypes: {
+    variant: {
+      control: { type: 'select' },
+      options: ['', 'danger'],
+      description: 'Adds a modifier class to the component to display the danger variation (see generated HTML)'
```

Comment:
Since this only has 2 options, switching this to a `boolean` control will make the JS more concise in the story. Also let's add a capitalized label to the controls panel by using the 'name' property in the argTypes object. (See Decision Card story argTypes object for "danger")

Reply:
Done